src/test/java/guitests/BoardAutoCreatorTest.java

```diff
@@ -105,14 +104,21 @@ public void sampleBoardAutoCreationTest() {
         waitUntilNodeAppears(SAMPLE_BOARD_DIALOG);
         click("OK");
 
-        assertEquals(panelControl.getPanelCount(), SAMPLE_PANEL_NAMES.size());
+        assertEquals(panelControl.getPanelCount(), BoardAutoCreator.getSamplePanelDetails().size());
         assertEquals(panelControl.getCurrentlySelectedPanel(), Optional.of(0));
         assertEquals(panelControl.getNumberOfSavedBoards(), 1);
 
         List<PanelInfo> panelInfos = panelControl.getCurrentPanelInfos();
-        for (int i = 0; i < SAMPLE_PANEL_NAMES.size(); i++){
-            assertEquals(panelInfos.get(i).getPanelFilter(), SAMPLE_PANEL_FILTERS.get(i));
-            assertEquals(panelInfos.get(i).getPanelName(), SAMPLE_PANEL_NAMES.get(i));
+        int i = 0;
+        for (String panelName : BoardAutoCreator.getSamplePanelDetails().keySet()) {
+            assertEquals(panelInfos.get(i).getPanelName(), panelName);
+            i++;
+        }
+
+        int j = 0;
+        for (String panelFilter : BoardAutoCreator.getSamplePanelDetails().values()) {
+            assertEquals(panelInfos.get(j).getPanelFilter(), panelFilter);
+            j++;
```

Comment:
We already have `panelInfo` class created. I'm making use of it as well. `BoardInfo` would include a list of panelInfos. This would result in quite a bit refactoring for the existing code on boards, which is beyond the scope of this PR. 

I am now using one loop to iterate through the map.